### PR TITLE
add log rotation configuration for phi log

### PR DESCRIPTION
**Pull request by egreer**
Fixes https://github.com/apsislabs/phi_attrs/issues/62 and allows rotation configuration to be passed in along with the path. Rotation is disabled by default.

For more information on the defaults see https://apidock.com/ruby/Logger/new/class

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ Example:
 ```ruby
 PhiAttrs.configure do |conf|
   conf.log_path = Rails.root.join("log", "phi_access_#{Rails.env}.log")
+  conf.log_shift_age  = 10 # how many logs to keep of `log_shift_size` or frequency to rotate (daily, weekly or monthly)
+  conf.log_shift_size = 100.megabyte # size in bytes when using `log_shift_age` as a number
```
Comment (HenryKeiter) [inline README.md:52]: I assume this needs to be plural

File: README.md [hunk 2]
```diff
@@ -48,6 +48,8 @@ Example:
 ```ruby
 PhiAttrs.configure do |conf|
   conf.log_path = Rails.root.join("log", "phi_access_#{Rails.env}.log")
+  conf.log_shift_age  = 10 # how many logs to keep of `log_shift_size` or frequency to rotate (daily, weekly or monthly)
```
Comment (HenryKeiter) [inline README.md:51]: Should include info about 0 == disabled here

File: spec/internal/config/initializers/phi_attrs.rb [hunk 3]
```diff
@@ -2,4 +2,8 @@
 
 PhiAttrs.configure do |conf|
   conf.log_path = Rails.root.join('log', 'phi_access.log')
+
+  # Log Rotation - disabled by default
+  conf.log_shift_age = 0  # how many logs to keep of `shift_size`
```
Comment (HenryKeiter) [inline spec/internal/config/initializers/phi_attrs.rb:7]: Should indicate here that 0 is "disabled"

File: README.md (hunk 1 above)
Comment (egreer) [reply README.md]: Either works in rails.

**Review by HenryKeiter — APPROVED** [review]

**Pull request merged**